tests/core/test_utilities.py

```diff
@@ -913,6 +914,53 @@ def test_fit_plane_to_points():
     )
 
 
+AIRPLANE_CASE = (
+    pv.examples.load_airplane().points,
+    [
+        [-8.12132213e-07, 9.92557054e-01, 1.21780514e-01],
+        [1.00000000e00, 8.09226640e-07, 7.33171741e-08],
+        [-2.57765578e-08, 1.21780514e-01, -9.92557054e-01],
+    ],
+)
+PLANAR_CASE = (
+    [[0, 0, 0], [1, 1, 1]],
+    [
+        [0.57735027, 0.57735027, 0.57735027],
+        [0.0, -0.70710678, 0.70710678],
+        [0.81649658, -0.40824829, -0.40824829],
+    ],
+)
```

Comment:
This is a rank 1, _linear_ case, right? So I'd call it `LINEAR_CASE`, and use a 3-point `PLANAR_CASE`. Assuming I understand the intent here correctly.

Reply:
Yes you are right it's two points (line) not a plane